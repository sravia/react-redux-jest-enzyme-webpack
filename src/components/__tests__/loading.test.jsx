import React from 'react';
import renderer from 'react-test-renderer';
import Loading from '../loading/Loading';

describe('<Loading />', () => {
  it('renders <Loading /> component', () => {
    const tree = renderer.create(<Loading />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
