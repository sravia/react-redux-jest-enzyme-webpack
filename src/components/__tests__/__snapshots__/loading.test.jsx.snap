// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders <Loading /> component 1`] = `
<div>
  <span>
    Loading...
  </span>
</div>
`;
